proto/config/appconfig.proto

```diff
@@ -206,6 +206,15 @@ message AppInstanceConfig {
 
   // PatchEnvelopeRef is used to track all patch envelopes used by the app instance
   PatchEnvelopeRef patchRef = 23;
+
+  // This edge-node UUID for the Designate Node for the Application
+  string designated_node_id = 24;
+
+  // If the App allows automatic migration
+  bool auto_migration = 25;
+
+  // a set of ResourceLabels for node selection
```

Comment:
remove the node_select_labels item for now, the first phase.

Reply:
ok, will remove this.

Reply:
removed